HappyAnding/HappyAnding/Views/Components/UserCurationCell.swift

```diff
@@ -72,6 +72,9 @@ struct UserCurationCell: View {
                     .padding(.bottom, 20)
                     .fixedSize(horizontal: false, vertical: true)
             }
+            .onAppear() {
+                curation.shortcuts = curation.shortcuts.sorted(by: { $0.title < $1.title})
```

Comment:
```suggestion
                curation.shortcuts.sort { $0.title < $1.title }
```
메모리 측면을 고려하면 sort함수를 고려하는 방법도 있을 것 같아요!
하지만 이 부분은 원본의 수정 가능성이 있을 수도 있으니 추후 리팩토링할 때 생각해보면 좋겠네요!!